messages/en.json

```diff
@@ -195,12 +195,12 @@
 	"massviews-empty-set": "$1 contains no pages!",
 	"massviews-external-link-description": "Get the pageviews of pages containing an $1",
 	"massviews-hashtag-description": "$1. Get the pageviews of pages that have edits with the given $2 in the edit summary",
-	"massviews-notice": "To view more than $1 pages, try $2.",
-	"massviews-oversized-set": "$1 contains $2 pages. For performance reasons, only the first $3 pages will be processed.",
-	"massviews-oversized-set-unknown": "$1 contains over $2 pages. For performance reasons, only the first $2 pages will be processed.",
+	"massviews-notice": "To view more than $1 {{PLURAL:$1|page|pages}}, try $2.",
+	"massviews-oversized-set": "$1 contains $2 {{PLURAL:$2|page|pages}}. For performance reasons, only the first $3 {{PLURAL:$3|page|pages}} will be processed.",
```

Comment:
So the issue here, and with some other messages, is that we're passing in a formatted number (e.g. `5,000`) instead of `5000`, which breaks the plural parser. This was a mistake, I should have passed in a dedicated parameter for us with PLURAL, like we do for [`num-bytes`](https://github.com/MusikAnimal/pageviews/blob/a2cf6f8ee24b22e96bf9888eb55b1bcf570cce17/messages/en.json#L228). I see that some translations are using PLURAL, so at this point it's probably best to do away with the formatting? I think it being appearing grammatically correct would be better than it being localized.

Reply:
You can still have both, it will just need a few more changes than just this file. Where you pass the number in the code, pass it twice – once formatted, and once unformatted. Then in these messages, just change it to $1 {{PLURAL:$2|page|pages}} where necessary.

Also: The number formatting is language-specific, yeah?

Reply:
Right, but the issue here is the order of the parameters. If I make `$3` the pluralized version, older outdated translations will show the wrong number with "For performance reasons, only first $3...". I almost want to create a new message altogether, or a Node script to convert the messages to be in the new format, though that would be quite a pain.

And yes, for this app JavaScript's [`toLocaleString`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toLocaleString) is used, which will format the number according to the user's browser and/or computer settings.

Reply:
What $number you use doesn't really matter. ☺️ You can have $5 {{PLURAL:$2|page|pages}} if you'd like, they can be rearranged for each language.